docs/src/main/sphinx/connector/hive.rst

```diff
@@ -606,6 +606,13 @@ Property Name                                        Description
 ``hive.metastore.glue.endpoint-url``                 Glue API endpoint URL (optional).
                                                      Example: ``https://glue.us-east-1.amazonaws.com``
 
+``hive.metastore.glue.sts.region``                   AWS region of the STS service to authenticate with. This is
```

Comment:
Do we need a new setting? why this cannot be based on `hive.metastore.glue.region`?

Reply:
It is necessary if we want to support situations like using Minio's STS with AWS Glue. 

There's also this: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html

TLDR: By default STS uses the global endpoint which is always available. You can set a region explicitly to reduce latency but STS is not available in all regions. So if we just use the Glue region but you're deployed in one of those places, for example `eu-south-1`, you need to use either the global endpoint or a diferent region with STS available.

I believe this means we need the setting.